index.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+require_once 'dbconn.php';
+require_once 'functions.php';
+
+$db = getDB();
+
+$games = getGames($db);
+
+?>
+
+<html lang="en-GB">
+    <head>
+        <title>Matt's Board Game Collection</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="preconnect" href="https://fonts.gstatic.com" />
+        <link href="https://fonts.googleapis.com/css2?family=Ubuntu&family=Yatra+One&display=swap" rel="stylesheet" />
+        <link href="normalize.css" type="text/css" rel="stylesheet" />
+        <link href="styles.css" type="text/css" rel="stylesheet" />
+    </head>
+    <body>
+        <nav>
+            <h1 tabindex="1">Matt's Board Game Collection</h1>
+            <button tabindex="1">Add Game</button>
+        </nav>
+        <main>
+            <?= displayGames($games); ?>
```

Comment:
to make the code more readable make a variable up the top that is the output frmo the display games function then gets output down here. just means can glance all of the functionality being called in the page in one place without having to hunt for it